components/sensor/ina219/ina219.go

```diff
@@ -0,0 +1,229 @@
+// Package ina219 implements an ina219 current/power monitor sensor
```

Comment:
Since this is meant to be used to detect low battery capacity conditions, let's add voltage to the list and state that battery monitoring is a key usecase for this sensor.